src/dashboard/cd-dashboard-stats.vue

```diff
@@ -1,30 +1,37 @@
 <template>
   <div class="column">
     <div class="cd-dashboard-stats">
-      <h1 class="cd-dashboard-stats__header">{{ $t('Dojo Stats') }}</h1>
-      <div class="cd-dashboard-stats__number">
-        <div class="cd-dashboard-stats__number-value"><span>{{ bookedChildren }}</span></div>
-        <div class="cd-dashboard-stats__description">{{ $t('kids attended your events') }}</div>
-      </div>
-      <div class="cd-dashboard-stats__pie">
-        <div class="cd-dashboard-stats__circle">
-          <svg viewBox="0 0 32 32">
-            <g v-for="gender in genderStats" >
-              <circle r="16" cx="16" cy="16" :style="{ strokeDashoffset: -gender.prevValue, strokeDasharray: `${gender.perc} 100`}"
-                :class="['cd-dashboard-stats__circle--' + gender.name]">
-              </circle>
-            </g>
-          </svg>
-          <div class="cd-dashboard-stats__circle-legends">
-            <div v-for="gender in genderStats" class="cd-dashboard-stats__circle-legend">
-              <div class="cd-dashboard-stats__circle-legend-color" :class="['cd-dashboard-stats__circle--' + gender.name]"></div>
-              <div stroke="#51c5cf" stroke-width="2px" text-anchor="middle" alignment-baseline="middle"> {{ gender.name }}</div>
+      <h3 class="cd-dashboard-stats__header">{{ $t('Dojo Stats') }}</h3>
+      <h4 class="cd-dashboard-stats__category">Youth</h4>
+      <div class="cd-dashboard-stats__charts" 
+        v-if="(bookedChildren === null || bookedChildren.length !== 0) && ((dojoUsers === null || dojoUsers.length !== 0) && (genders === null || genders.length !== 0))">
+        <div class="cd-dashboard-stats__chart-number" v-if="bookedChildren !== null">
+          <span class="cd-dashboard-stats__description" v-html="$t('{nbKids} kids attended your events', { nbKids: numberStatText })"></span>
```

Comment:
`{nbKids}` wouldn't be very obvious to a translator. Just `{num}` or `{numKids}` would be better.